modules/apps/document-library/document-library-web/src/main/java/com/liferay/document/library/web/internal/display/context/logic/UIItemsBuilder.java

```diff
@@ -427,6 +430,19 @@ public DropdownItem createMoveDropdownItem() {
 			(_fileShortcut != null) ?
 				String.valueOf(_fileShortcut.getFileShortcutId()) :
 					String.valueOf(_fileEntry.getFileEntryId())
+		).putData(
+			"selectFolderURL",
+			() -> {
+				FolderItemSelectorURLProvider folderItemSelectorURLProvider =
+					new FolderItemSelectorURLProvider(
+						_httpServletRequest,
+						(ItemSelector)_httpServletRequest.getAttribute(
```

Comment:
this causes :
```
ERROR [http-nio-8080-exec-1][IncludeTag:119] Current URL /group/guest/~/control_panel/manage?p_p_id=com_liferay_document_library_web_portlet_DLAdminPortlet&p_p_lifecycle=2&p_p_state=maximized&p_p_mode=view&p_p_resource_id=%2Fdocument_library%2Finfo_panel&p_p_cacheability=cacheLevelPage&_com_liferay_document_library_web_portlet_DLAdminPortlet_folderId=32938&_com_liferay_document_library_web_portlet_DLAdminPortlet_repositoryId=20117&p_p_auth=5aiuoPBP&_com_liferay_document_library_web_portlet_DLAdminPortlet_priv_r_p_mvcRenderCommandName=%2Fdocument_library%2Fview_folder&_com_liferay_document_library_web_portlet_DLAdminPortlet_priv_r_p_redirect=http%3A%2F%2Flocalhost%3A8080%2Fgroup%2Fguest%2F%7E%2Fcontrol_panel%2Fmanage%3Fp_p_id%3Dcom_liferay_document_library_web_portlet_DLAdminPortlet%26p_p_lifecycle%3D0%26p_p_state%3Dmaximized%26p_p_mode%3Dview%26p_p_auth%3D5aiuoPBP generates exception: java.lang.RuntimeException: java.lang.RuntimeException: java.lang.NullPointerException
java.lang.NullPointerException: null
	at com.liferay.document.library.web.internal.util.FolderItemSelectorURLProvider._getItemSelectorURL(FolderItemSelectorURLProvider.java:119) ~[bundleFile:?]
	at com.liferay.document.library.web.internal.util.FolderItemSelectorURLProvider.getSelectMoveToFolderURL(FolderItemSelectorURLProvider.java:77) ~[bundleFile:?]
	at com.liferay.document.library.web.internal.display.context.logic.UIItemsBuilder.lambda$createMoveDropdownItem$2(UIItemsBuilder.java:442) ~[bundleFile:?]
	at com.liferay.frontend.taglib.clay.servlet.taglib.util.DropdownItemBuilder$DropdownItemStep.putData(DropdownItemBuilder.java:199) ~[bundleFile:?]
	at com.liferay.document.library.web.internal.display.context.logic.UIItemsBuilder.createMoveDropdownItem(UIItemsBuilder.java:433) ~[bundleFile:?]
	at com.liferay.document.library.web.internal.display.context.DefaultDLViewFileVersionDisplayContext.lambda$getActionDropdownItems$1(DefaultDLViewFileVersionDisplayContext.java:141) ~[bundleFile:?]
	at com.liferay.frontend.taglib.clay.servlet.taglib.util.DropdownItemList.addGroup(DropdownItemList.java:106) ~[bundleFile:?]
	at com.liferay.frontend.taglib.clay.servlet.taglib.util.DropdownItemListBuilder$DropdownItemListWrapper.addGroup(DropdownItemListBuilder.java:272) ~[bundleFile:?]
	at com.liferay.document.library.web.internal.display.context.DefaultDLViewFileVersionDisplayContext.getActionDropdownItems(DefaultDLViewFileVersionDisplayContext.java:119) ~[bundleFile:?]
	at com.liferay.document.library.display.context.BaseDLViewFileVersionDisplayContext.getActionDropdownItems(BaseDLViewFileVersionDisplayContext.java:45) ~[bundleFile:?]
	at com.liferay.document.library.opener.onedrive.web.internal.display.context.DLOpenerOneDriveDLViewFileVersionDisplayContext.getActionDropdownItems(DLOpenerOneDriveDLViewFileVersionDisplayContext.java:90) ~[?:?]
	at com.liferay.document.library.display.context.BaseDLViewFileVersionDisplayContext.getActionDropdownItems(BaseDLViewFileVersionDisplayContext.java:45) ~[bundleFile:?]
	at com.liferay.document.library.opener.google.drive.web.internal.display.context.DLOpenerGoogleDriveDLViewFileVersionDisplayContext.getActionDropdownItems(DLOpenerGoogleDriveDLViewFileVersionDisplayContext.java:91) ~[?:?]
	at com.liferay.document.library.display.context.BaseDLViewFileVersionDisplayContext.getActionDropdownItems(BaseDLViewFileVersionDisplayContext.java:45) ~[bundleFile:?]
	at com.liferay.sharing.document.library.internal.display.context.SharingDLViewFileVersionDisplayContext.getActionDropdownItems(SharingDLViewFileVersionDisplayContext.java:74) ~[?:?]
	at org.apache.jsp.document_005flibrary.file_005fentry_005faction_jsp._jspService(file_005fentry_005faction_jsp.java:493) ~[?:?]
	at org.apache.jasper.runtime.HttpJspBase.service(HttpJspBase.java:111) ~[jasper.jar:9.0.83]
	at javax.servlet.http.HttpServlet.service(HttpServlet.java:623) ~[servlet-api.jar:4.0.FR]
	at org.apache.jasper.servlet.JspServletWrapper.service(JspServletWrapper.java:411) ~[javax.servlet.jsp-2.3.3-b02.LIFERAY-PATCHED-9.jar:9.0.83]
	at org.apache.jasper.servlet.JspServlet.serviceJspFile(JspServlet.java:473) ~[javax.servlet.jsp-2.3.3-b02.LIFERAY-PATCHED-9.jar:9.0.83]
```

Reply:
solved on https://github.com/liferay-lima/liferay-portal/pull/5030/commits/dc2682e4a7b46c8becb701149177563fb22b6f92

Please review